src/Stratis.Bitcoin.Features.Wallet/Controllers/WalletController.cs

```diff
@@ -668,14 +669,16 @@ public IActionResult SendTransaction([FromBody] SendTransactionRequest request)
                     Outputs = new List<TransactionOutputModel>()
                 };
 
-                foreach (var output in transaction.Outputs)
+                foreach (var output in transaction.Outputs.Where(txOut => !txOut.ScriptPubKey.IsUnspendable))
```

Comment:
I'd rather see everything inside the foreach.
You can loop around the outputs and if the output is unspendable, a `TransactionOutputModel` for it is added, otherwise a normal one is added.